addons/mail/static/src/components/composer/composer.xml

```diff
@@ -98,25 +98,27 @@
                             </t>
                             <div class="o_Composer_primaryToolButtons" t-att-class="{ 'o-composer-is-compact': props.isCompact }">
                                 <Popover position="'top'" t-on-o-emoji-selection="_onEmojiSelection">
-                                    <!-- TODO FIXME o-open not possible to code due to https://github.com/odoo/owl/issues/693 -->
+                                    <!-- TODO FIXME o-open and aria-expanded not possible to code due to https://github.com/odoo/owl/issues/693 -->
```

Comment:
Is there any issue created for this, if so, it might be good to add the link or issue # here so it doesn't go under the radar.

Reply:
It is already done in master, it should be handled immediately at forward-port.